data/transactions/logic/doc.go

```diff
@@ -111,6 +118,23 @@ var opDocByName = map[string]string{
 	"asset_holding_get": "read from account specified by Txn.Accounts[A] and asset B holding field X (imm arg) => {0 or 1 (top), value}",
 	"asset_params_get":  "read from asset Txn.ForeignAssets[A] params field X (imm arg) => {0 or 1 (top), value}",
 	"assert":            "immediately fail unless value X is a non-zero number",
+	"callsub":           "branch unconditionally to TARGET, saving the next instruction on the call stack",
+	"retsub":            "pop the top instruction from the call stack and branch to it",
+
+	"b+":  "A plus B, where A and B are byteslices interpreted as big-endian unsigned integers",
+	"b-":  "A minus B, where A and B are byteslices interpreted as big-endian unsigned integers. Panic on underflow.",
+	"b/":  "A divided by B, where A and B are byteslices interpreted as big-endian unsigned integers. Panic if B is zero.",
+	"b*":  "A times B, where A and B are byteslices interpreted as big-endian unsigned integers.",
+	"b<":  "A is less than B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b>":  "A is greater than B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b<=": "A is less than or equal to B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b>=": "A is greater than or equal to B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b==": "A is equals to B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b!=": "A is not equal to B, where A and B are byteslices interpreted as big-endian unsigned integers => { 0 or 1}",
+	"b%":  "A modulo B, where A and B are byteslices interpreted as big-endian unsigned integers. Panic if B is zero.",
+	"b|":  "A bitwise-or B, where A and B are byteslices interpreted as big-endian unsigned integers.",
+	"b&":  "A bitwise-and B, where A and B are byteslices interpreted as big-endian unsigned integers.",
+	"b^":  "A bitwise-xor B, where A and B are byteslices interpreted as big-endian unsigned integers.",
```

Comment:
Why not have `b~` for bitwise invert as well?

Reply:
Saying all this outloud led to a better way.  bitwise operation no longer truncate to the shortest byteslice that can hold the value.  They return a byteslice as long as their longest input.  This is more likely to be what people want, as they would use these operations when treating the slices as arrays of bits.  It also makes the definition of b~ more straightforward.